.travis.yml

```diff
@@ -3,7 +3,14 @@ language: node_js
 node_js:
   - 12.4.0
 
-before_install:
+services:
+  - docker
+
+before_install: 
+  - echo minor > versionType.txt
+  - echo Test > destination.txt
+  - docker build . --tag="nwisweb_tableau_dockerinstance"
+  - docker run --rm --name nwisweb_data_connector_docker nwisweb_tableau_dockerinstance
```

Comment:
So, if all of this moving into a Docker container, do you still need to do the `npm install` on the next line?

Reply:
Yeah you are completely right about that